src/components/illustrations/XPIcon.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, useTheme } from '@mui/material';
+
+// XPIcon
+interface XPIconProps {
+  size?: number;
+}
+
+export const XPIcon = ({ size }: XPIconProps) => {
```

Comment:
if the icon code is the same maybe it is better to change just colors based on theme?